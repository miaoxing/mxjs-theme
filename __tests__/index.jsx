import {ThemeProvider, useTheme} from '../index';
import {render} from '@testing-library/react';

describe('theme', () => {
  test('basic', () => {
    const Child = () => {
      const theme = useTheme();
      return theme.key;
    };

    const {container} = render(<ThemeProvider theme={{
      key: 'value',
    }}>
      <Child/>
    </ThemeProvider>);
    expect(container).toMatchSnapshot();
  });
});
