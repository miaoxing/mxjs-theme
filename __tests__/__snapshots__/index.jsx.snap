// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`theme basic 1`] = `
<div>
  value
</div>
`;
